frontend/openchat-shared/src/domain/crypto.ts

```diff
@@ -130,3 +103,8 @@ export type AccountTransactions = {
 };
 
 export type AccountTransactionResult = Failure | (Success & AccountTransactions);
+
+export type TokenExchangeRates = {
+    to_icp: number | undefined;
```

Comment:
Slightly eccentric casing for js-land

Reply:
To much going back and forth between languages!